phpBB/phpbb/controller/helper.php

```diff
@@ -71,7 +71,8 @@ class helper
 	* @param \phpbb\template\template $template Template object
 	* @param \phpbb\user $user User object
 	* @param \phpbb\config\config $config Config object
-	* @param \phpbb\controller\provider $provider Path provider
+	 *
```

Comment:
You have indented these with PHPStorm apparently. However, the indentation on these two lines is different than the other lines before and after it.